cpu/esp32/Makefile.include

```diff
@@ -165,7 +164,8 @@ FLASHFILE ?= $(ELFFILE)
 FLASH_MODE ?= dout  # FIX configuration, DO NOT CHANGE
 FLASH_FREQ  = 40m   # FIX configuration, DO NOT CHANGE
 FLASH_SIZE ?= 2MB
-PREFLASHER = $(ESPTOOL)
+
+PREFLASHER = $(RIOTBASE)/dist/tools/esptool/esptool.py
```

Comment:
I would not do it this way, but rather only change line 67 as follows:
```mk
ESPTOOL ?= $(RIOTBASE)/dist/tools/esptool/esptool.py
```

This way it's still possible to easily override the ESPTOOL path from the command line.

Reply:
Ok, I just did it in that way to be consistent with `cpu/esp8266/Makefile.include` because both makefiles will be merged in `cpu/esp_common`Makefile.include` in short term. I will change it.